packages/next/src/client/lib/is-chrome.ts

```diff
@@ -0,0 +1,11 @@
+export function isChrome() {
+  if (typeof window === 'undefined') return false
+  const isChromium = 'chrome' in window && window.chrome
+  const vendorName = window.navigator.vendor
+
+  return (
+    isChromium !== null &&
+    isChromium !== undefined &&
+    vendorName === 'Google Inc.'
+  )
+}
```

Comment:
I'm still not convinced by this. This needs to be 100% accurate. 

Let's at least inline it so that people don't blindly use it and make sure it's only used as a hint and not requirement for using the copy button.